api/tests/integration/edge_api/identities/test_edge_identity_featurestates_viewset.py

```diff
@@ -694,10 +694,7 @@ def test_edge_identities_post_returns_400_for_invalid_mvfs_allocation(
         identity_uuid
     )
     assert response.status_code == status.HTTP_400_BAD_REQUEST
-    assert (
-        response.json()["multivariate_feature_state_values"]
-        == "Total percentage allocation for feature must be less than 100 percent"
-    )
+    assert "multivariate_feature_state_values" in response.json()
```

Comment:
How come this was changed?

Reply:
The message was changed in flagsmith-engine and this broke the test. I decided not to test the library code.